src/symbol/symbol_layout.js

```diff
@@ -510,7 +550,11 @@ function addSymbol(bucket: SymbolBucket,
     let numVerticalGlyphVertices = 0;
     const placedTextSymbolIndices = {};
     let key = murmur3('');
-    const radialTextOffset = (layer.layout.get('text-radial-offset').evaluate(feature, {}) || 0) * ONE_EM;
+    let radialTextOffset0 = (layer.layout.get('text-radial-offset').evaluate(feature, {}) || 0) * ONE_EM;
+    let radialTextOffset1 = Number.POSITIVE_INFINITY;
+    if (!radialTextOffset0) {
```

Comment:
Would it be better to check for undefined? Right now, undefined and `text-radial-offset: 0` behave in the same way, falling back to `text-offset`.

Reply:
Fixed in the latest commit, thanks!